### DISCUSSION TITLE
Anyway to import Youtube Playlist?

### DISCUSSION THREAD

**Discussion by BryanChung** [z-huang/music#523] [Q&A] [answered]
I did a search on Discussions and it says I needed to download a Debug version? But I cannot find it. Can kindly link me to it or it's already in the main app now? (Since that discussion was last updated on August 2022).

**Comment by z-huang** [comment, marked as answer]
Make your playlist public, and search it in InnerTune.

**Comment by BryanChung** [comment]
Ah that works too! Thank you!

So when the Youtube playlist gets updated with new song, innertube will also get updated?

**Reply by z-huang** [reply]
Nope.

**Reply by BryanChung** [reply]
So I have to always remove playlist, add playlist?

**Reply by z-huang** [reply]
You have two options:
1. Add to library: create a link to the online playlist. The playlist won't be saved in local, but you'll always see the latest playlist.
2. Import: create a local playlist and add your online playlist songs to it. You can view it when offline.

**Reply by BryanChung** [reply]
Wow that's really awesome! Going to try! THANKS! :D

**Comment by kovachzg** [comment]
i imported my playlist in YTM, and made it public. I gave it some name, but i can't seem to find it trough InnerTune search/Featured playlists. Does it take some time to YT to propagate it publicly or ?

**Reply by owenconnorz** [reply]
Login into your account using the debut version this will find it so much quicker @kovachzg
